lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json

```diff
@@ -47,8 +47,9 @@
     {
       "name": "iOSScrollBehavior",
       "type": "number",
-      "description": "The scrolling behavior, use KeyboardAccessoryView.iosScrollBehaviors.X where X is:\nNONE, SCROLL_TO_BOTTOM_INVERTED_ONLY or FIXED_OFFSET\niOS only.",
-      "default": "FIXED_OFFSET"
+      "description": "The scrolling behavior. DEPRECATED: Use 'scrollBehavior' prop instead and take values from 'KeyboardTrackingView.scrollBehaviors' (instead of KeyboardAccessoryView.iosScrollBehaviors)",
```

Comment:
This will need to change as well if we decide to keep the static on `KeyboardAccessoryView`

Reply:
Yes, keeping the static. fixed.